GoogleForCreators/web-stories-wp
# Quick Actions: Open style tab after adding a text element

## Context


## Summary



## Relevant Technical Choices
Previously, the highlight was on the text sets panel which doesn't really make sense since it's an insertion panel for text sets and not related to styling the newly added text element.


## To-do



## User-facing changes



## Testing Instructions






This PR can be tested by following these steps:

1. Unselect all elements
2. Click the quick-action to insert text.
3. Verify the Style tab gets opened
4. Try again the same to verify it works the second time, too


## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #11291
